src/main/java/christmas/domain/event/VisitDate.java

```diff
@@ -0,0 +1,40 @@
+package christmas.domain.event;
+
+import christmas.utils.DayOfWeekParser;
+import java.time.DayOfWeek;
+
+public class VisitDate {
+
+    private static final int MIN_ORDER_DATE = 1;
+    private static final int MAX_ORDER_DATE = 31;
+
+    private final int date;
+
+    private VisitDate(int date) {
+        this.date = date;
+    }
+
+    public static VisitDate from(int date) {
+        validateDateRange(date);
+        return new VisitDate(date);
+    }
+
+    private static void validateDateRange(int date) {
+        if (!meetsDateRangeCriterion(date)) {
+            throw new IllegalArgumentException("date %d is in invalidate date range.".formatted(date));
+        }
+    }
+
+    private static boolean meetsDateRangeCriterion(int date) {
+        return date >= MIN_ORDER_DATE && date <= MAX_ORDER_DATE;
+    }
+
+    public int getDate() {
+        return date;
+    }
+
+    public DayOfWeek getDayOfWeek() {
+        return DayOfWeekParser.parse2023DecemberByDate(this.date);
```

Comment:
(개인적인 생각입니다!)
```suggestion
        return DayOfWeekParser.parse2023DecemberByDate(this.date);
```
getDayOfWeek 메서드가 호출되는 상황에서 LocalDate가 생성되는데 오히려 멤버 변수를 LocalDate타입이었다면 반복적으로 생성하지 않아도 될 것 같습니다 :)